src/transformState.js

```diff
@@ -46,7 +46,28 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
-}
+  for (const element in transforms) {
+    const props = transforms[element]['properties'];
```

Comment:
Пиши свойства через точку
```suggestion
    const properties = transforms[element].properties;
```
Делай деструктуризацию
```suggestion
    const { properties } = transforms[element];
```